actors/verifreg/src/lib.rs

```diff
@@ -767,6 +919,18 @@ where
     )
 }
 
+fn can_claim_alloc(claim_alloc: &SectorAllocationClaim, alloc: &Allocation, curr_epoch: ChainEpoch) -> bool {
+    let sector_lifetime = claim_alloc.sector_expiry - curr_epoch;
+
+    claim_alloc.provider == alloc.provider
+    && claim_alloc.client == alloc.client 
+    && claim_alloc.piece_cid == alloc.data
+    && claim_alloc.piece_size == alloc.size 
+    && curr_epoch <= alloc.expiration
+    && sector_lifetime >= alloc.term_min
+    && sector_lifetime < alloc.term_max
```

Comment:
I think this one can be `<=` tho. 

If sector activated during epoch 0 and expires during epoch 100, then it has 100 epochs of life (epochs 1 thru 100 inclusive). If current epoch is 0 and term max is 100, then the allocation can be claimed by above sector.